### DISCUSSION TITLE
Contribution Resources For Rust/Flutter Novice

### DISCUSSION THREAD

**Discussion by coder0xff** [fzyzcjy/flutter_rust_bridge#1009] [Q&A]
Hi. I'd like to fork this repository, make some edits, and build and test my changes on my local machine, using an ongoing flutter_rust_bridge_template project as my test bed. As someone who's new to Rust and to Dart/Flutter, the workflow to do that is unfamiliar to me. I have lots of questions, for instance, can I install a editable package in Cargo the same way I can `pip install -e` in Python? What's scoop.json.sh? If you could share a bit about the workflow, or point me to online resources that will help me get started, I would really appreciate it. Thanks!

**Comment by fzyzcjy** [comment]
> can I install a editable package in Cargo the same way I can pip install -e in Python

Just install using path dependency, which is (IMHO) much more convenient than py ;)

> What's scoop.json.sh?

for release, not development

> If you could share a bit about the workflow, or point me to online resources that will help me get started, I would really appreciate it.

Maybe google how to develop a standard rust package, a standard flutter package

**Reply by coder0xff** [reply]
I see there's a test suite that runs in GitHub, but it looks like integration tests instead of unit tests. How can I debug an invocation of flutter_rust_bridge_codegen in VS Code that will allow me to use breakpoints, inspect variables, etc?

**Reply by coder0xff** [reply]
Never mind, I found the `#[test]`s in the Rust files.
